Sasquatch/Sasquatch/ViewControllers/MSCustomPropertiesViewController.swift

```diff
@@ -132,7 +132,7 @@ class MSCustomPropertiesViewController : UITableViewController, AppCenterProtoco
       tableView.deleteRows(at: [indexPath], with: .automatic)
     } else if editingStyle == .insert {
       let count = properties.count
-      properties.insert(("key\(count)", CustomPropertyType.String, "value\(count)"), at: 0)
+      properties.insert(("key\(count)", CustomPropertyType.Clear, "value\(count)"), at: 0)
```

Comment:
I'm not sure why default `CustomPropertyType` was changed from `Clear` to `String`.
If you have any objections about this change – let's discuss them =)

Reply:
Yep. Please revert and update the tests instead